app/assets/javascripts/startpage/surveyQuestions.coffee

```diff
@@ -96,13 +96,15 @@ surveyQuestions.config(["$routeProvider", ($routeProvider) ->
 
     # Asynchronously load the list of questions for a topic
     load_questions = (topicId) ->
-        SharedRequests.requestQuestionsByTopic(topicId).success( (allQuestions) ->
-            $scope.questions = []
-            allQuestions = allQuestions.sort((u, v) -> u.index - v.index)
-            for question in allQuestions
-                $scope.questions.push(question)
-            $scope.numQuestions = $scope.questions.length
-        )
+        if $scope.questions.length == 0
+            SharedRequests.requestQuestionsByTopic(topicId).success( (allQuestions) ->
+                $scope.questions = []
```

Comment:
just code style, but you can do $scope.questions = allQuestions.sort (u, v) -> u.index - v.index